cli/parse_pdfs.py

```diff
@@ -351,16 +363,27 @@ def parse_file(
                 )
                 all_pages_metadata.append(page_metadata)
                 continue
-            _LOGGER.info(f"Running postprocessing for page {page_idx}")
+            _LOGGER.info(f"Running google document structure OCR for page {page_idx}")
+            google_layout = extract_google_layout(image)[1]
+            # Combine the Google text blocks with the layoutparser layout.
+            combined_layout = combine_google_lp(
+                image,
+                google_layout,
+                layout_disambiguated,
+                threshold=replace_threshold,
+                top_exclude=top_exclude_threshold,
+                bottom_exclude=bottom_exclude_threshold,
+            )
```

Comment:
Why not add "parameter=variable" for all rather than just "variable" for the top 3?

Reply:
Not sure I'm following here.

Reply:
Just out of interest why not the following as parameters:

**image=image,
google_layout=google_layout,
layout=layout_disambiguated,**
threshold=replace_threshold,
top_exclude=top_exclude_threshold,
bottom_exclude=bottom_exclude_threshold,